examples/custom-clone.js

```diff
@@ -0,0 +1,37 @@
+export const g = {"Queue": null, "Link": null, "stringify": null, "Button": null,
+                  "createElement": null, "useRecordContext": null, "useCreatePath": null}
+
+const CopyUSButton = (props) => {
+    const {
+        label = "Copy to US",
+        scrollToTop = true,
+        icon = g.createElement(g.Queue),
```

Comment:
Then we use the functions from the global object the same as we would if we had imported them normally.

We can't use JSX here, because it will evaluate to `null` at compile time and not work at all.